beta/nncf/tensorflow/quantization/algorithm.py

```diff
@@ -72,25 +74,74 @@ def __init__(self, config):
         self.ignored_scopes_per_group = {}
         self.target_scopes_per_group = {}
 
-        for quantizer_group in QUANTIZER_GROUPS:
+        for quantizer_group in QuantizerGroup:
             self._parse_group_params(self.config, quantizer_group)
 
-    def build_controller(self, model):
-        return QuantizationController(model, self.config)
-
-    def _parse_group_params(self, config, quantizer_group):
-        params_dict = config.get(quantizer_group, {})
-        self.global_quantizer_constraints[quantizer_group] = QuantizationConstraints(
-            num_bits=params_dict.get('bits'),
-            mode=params_dict.get('mode'),
-            signedness_to_force=params_dict.get('signed'),
-            per_channel=params_dict.get('per_channel')
-        )
+        if self.should_init:
+            self._parse_init_params()
+            self._initializer = MinMaxInitializer(self._range_init_params)
+
+    def _parse_init_params(self):
+        init_config = self.config.get('initializer', {})
+        self._range_init_params = self._parse_range_init_params(init_config)
+
+    def _parse_range_init_params(self, initializer_config: Dict) -> RangeInitParams:
```

Comment:
Is it a common code? If it is possible I think it is better to move this code to the common part.
If not, can we create a separate module for TF only that extracts params from the config?

Reply:
Thanks for the comment!